### PR TITLE
Auth1

### PR DESCRIPTION
**Pull request by vmatyukh**

### CODE REVIEW

File: client/src/chess/auth/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  ngRoute
+]).
+config(RouteConfig).
+component('auth', AuthComponent).
+factory('auth', AuthFactory);
+
+RouteConfig.$inject = ['$routeProvider'];
+function RouteConfig($routeProvider) {
+  $routeProvider.when('/login', {
+		template: '<auth>'
```
Comment (tkustov) [inline client/src/chess/auth/index.js]:
```
<auth></auth>
```

File: lib/server/auth/models/db.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+var mongoose = require( 'mongoose' );
+var dbURI = 'mongodb://127.0.0.1:27017/test';
```
Comment (tkustov) [inline lib/server/auth/models/db.js]: dbUrl should be an environment variable (like API_URL)

File: client/src/chess/startup/index.js [hunk 3]
```diff
@@ -8,7 +8,29 @@ config(RouteConfig);
...
+  .when('/', {
     templateUrl: 'startup/view.html'
+  })
+  .when('/chess', {
+    controller: CheckAuth,
+    templateUrl: 'startup/chess.html'
   });
 }
+
+CheckAuth.$inject = ['$http', '$location']
```
Comment (tkustov) [inline client/src/chess/startup/index.js]: check for user authentication using $http interceptors

File: client/src/chess/auth/index.js (hunk 1 above)
Comment (vmatyukh) [inline client/src/chess/auth/index.js]: Done

File: client/src/chess/startup/index.js (hunk 3 above)
Comment (vmatyukh) [inline client/src/chess/startup/index.js]: Done

File: lib/server/auth/models/db.js (hunk 2 above)
Comment (vilmesh) [inline lib/server/auth/models/db.js]: done

**Pull request merged**